source/Various/LawvereFPT.lagda

```diff
@@ -191,6 +197,67 @@ As a simple application, it follows that negation doesn't have fixed points:
 
 \end{code}
 
+
+Assuming function extensionality, propositional extensionality, and
+propositional resizing, we may further prove the statement of Cantor's theorem
+from embeddings. Our argument follows Taylor's Practical Foundations of
+Mathematics, via the nLab: https://ncatlab.org/nlab/show/Cantor%27s+theorem.
+
+\begin{code}
+
+ cantor-theorem-for-embeddings
+  : FunExt
+  → PropExt
+  → Propositional-Resizing
+  → (A : 𝓤 ̇ )
+  → (ϕ : (A → Ω₀) → A)
```

Comment:
I'd like to state this with `\Omega U` instead; this makes the inside of the proof a bit more complex, but not impossibly so.